satpy/readers/landsea_nc.py

```diff
@@ -0,0 +1,180 @@
+        """
+
+        for key, val in variable.attrs.items():
+            if isinstance(val, h5py._hl.base.Empty):
+                variable.attrs.pop(key)
+
+        if '_fill_value' in variable.attrs:
+            variable = variable.where(variable != variable.attrs['_fill_value'])
+
+        if 'valid_range' in variable.attrs:
+            variable = variable.where(
+                variable <= variable.attrs['valid_range'][1])
+
+        attrs = variable.attrs.copy()
+
+        variable.attrs = attrs
+
+        variable.attrs.update({'platform_name': self.platform_name,
+                               'sensor': self.sensor})
+        if 'units' in variable.attrs:
+            variable.attrs.setdefault('units', str(variable.units))
+        else:
+            variable.attrs.setdefault('units', '1')
+
+        return variable
+
+    def upsample_geolocation(self, dsid, info):
+        """Upsample the geolocation (lon,lat) from the tiepoint grid."""
+        from geotiepoints import SatelliteInterpolator
+        # Read the fields needed:
+        col_indices = self.nc['nx_reduced'].values
+        row_indices = self.nc['ny_reduced'].values
+        lat_reduced = self.scale_dataset(dsid, self.nc['lat_reduced'], info)
+        lon_reduced = self.scale_dataset(dsid, self.nc['lon_reduced'], info)
+
+        shape = (self.nc['y'].shape[0], self.nc['x'].shape[0])
+        cols_full = np.arange(shape[1])
+        rows_full = np.arange(shape[0])
+
+        satint = SatelliteInterpolator((lon_reduced.values, lat_reduced.values),
+                                       (row_indices,
+                                        col_indices),
+                                       (rows_full, cols_full))
+
+        lons, lats = satint.interpolate()
+        self.cache['lon'] = xr.DataArray(lons, attrs=lon_reduced.attrs, dims=['y', 'x'])
+        self.cache['lat'] = xr.DataArray(lats, attrs=lat_reduced.attrs, dims=['y', 'x'])
+
+        return
+
+    def get_area_def(self, dsid):
+        """Get the area definition of the datasets.
+           fixed area definition worldeqc4km21
+        """
+        area = get_area_def("worldeqc4km21",
+                            "World 7200 x 3600, platecarree",
+                            'eqc',
+                            '+ellps=WGS84 +lat_0=0 +lat_ts=0 +lon_0=0 +no_defs' +
+                            ' +proj=eqc +type=crs +units=m +x_0=0 +y_0=0',
+                            7200,
+                            3600,
+                            (-20037508.3428, -10018754.1714, 20037508.3428, 10018754.1714))
+
+        return area
+
+    def __del__(self):
+        """Delete the instance."""
+        if self._unzipped:
+            try:
+                os.remove(self._unzipped)
+            except (IOError, OSError):
+                pass
+
+    @property
+    def start_time(self):
+        """Return the start time of the object."""
+        return datetime.strptime("2007-06-28 00:00:00", "%Y-%m-%d %H:%M:%S")
```

Comment:
I suppose this isn't provided in the netcdf file ?

Reply:
Yes, there is no date in the file. 
If you like to test the function, I uploaded the file here:
https://drive.google.com/file/d/1Bu37sxz8h39MsO_lGN9KOVFXQEnuQzB-/view?usp=sharing